CMakeLists.txt

```diff
@@ -48,7 +49,15 @@ project(ccl VERSION 0.2.1)
     else(EXTERNAL_CLASS_PATH)
       include(BuildCLASS)
     endif(EXTERNAL_CLASS_PATH)
-
+    
+    # Check whether an older version of CLASS is being used
+    if(CLASS_VERSION_PRE27)
+      message("Using an old (pre-v2.7.0) version of CLASS")
+      message("  CLASS_VERSION_PRE27: ${CLASS_VERSION_PRE27}")
+    else(CLASS_VERSION_PRE27)
+      message("Assuming CLASS v2.7.0 or later")
```

Comment:
The class devs clearly do not know about semantic versioning. Why is 2.6 to 2.7 an API change?!

Reply:
INDEED!